package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint normandy client webpack.config.js karam.conf.js"
```

Comment:
I think you're right. I made it `npm run lint` so linting was standardized, but the output kind of sucks. I think the JUnit output makes this ok though? Also I'd have to do `node_modules/.bin/eslint client ...` because PATH issues.

Why `|| true`? That makes the command always pass, which I don't think is what we want.